src/Umbraco.Web.UI.Client/src/views/member/apps/content/content.controller.js

```diff
@@ -5,6 +5,14 @@
 
         var vm = this;
 
+        vm.hideSystemProperties = function (property) {
+            // hide some specific, known properties by alias
+            if (property.alias === "_umb_id" || property.alias === "_umb_doctype") {
+                return false;
+            }
+            // hide all label properties with the alias prefix "umbracoMember" (e.g. "umbracoMemberFailedPasswordAttempts")
+            return property.view !== "readonlyvalue" || property.alias.indexOf("umbracoMember") !== 0;
```

Comment:
Hi @KennethJakobsen, this is me being picky, but I'd use `!property.alias.startsWith('umbracoMember')` rather than `indexOf`, for those extra readability points. Like I said, me being picky and preferring the natural language function names 😄 Is there a JS style guide for the UI?

Reply:
Even more picky, we like using `=== false` instead of the easily overlooked `!` in front of something 😅 

```suggestion
            return property.view !== "readonlyvalue" || property.alias.startsWith('umbracoMember') === false;
```

There is no JS style guide, yet ... ;-)